src/main/java/org/apache/commons/codec/binary/BinaryCodec.java

```diff
@@ -136,8 +136,10 @@
      * @param array
      *            the source array
      * @return {@code true} if the given array is {@code null} or empty (size 0.)
+     *
+     * @since 1.16 change visibility to public
      */
-    private static boolean isEmpty(final byte[] array) {
+    public static boolean isEmpty(final byte[] array) {
```

Comment:
Hello @arturobernalg 
Please make this package private instead of public, this helps minimize the API footprint we will need to maintain for BC, especially since this is just for internal use.